bundle/src/main/java/com/adobe/acs/commons/replication/status/impl/JcrPackageReplicationStatusEventHandler.java

```diff
@@ -231,7 +245,7 @@ public final JobResult process(final Job job) {
 
                     if (resources.size() > 0) {
                         replicationStatusManager.setReplicationStatus(resourceResolver,
-                                this.replicatedBy,
+                                StringUtils.defaultIfEmpty(replicatedBy, "Package Replication"),
```

Comment:
Why do you need a fallback here as well, the property `PROPERTY_REPLICATED_BY` is never null/empty.